pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -905,6 +905,8 @@ func (qjm *XController) getAggregatedAvailableResourcesPriority(unallocatedClust
 			continue
```

Comment:
I don't think we need treat the target AW differently. There could still be pods associated with a requeued AW. We should invoke `addTotalSnapshotResourcesConsumedByAw` as in the next case (`!value.Status.CanRun`) as the intent is for these pods to terminate.

Reply:
In other words, we can probably delete this case, but this is probably not a must fix.

Reply:
This is about target AW (meaning from all AWs in the system if we got AW that is same as target AW we ignore) is not dispatched so it needs to be ignore.